docs/make.jl

```diff
@@ -61,16 +61,18 @@ makedocs(
                 "Group manifold" => "manifolds/group.md",
             ],
         ],
-        "Statistics" => "statistics.md",
-        "Distributions" => "distributions.md",
-        "About" => "about.md",
-        "Contributing" => "generated/contributing.md",
-        "Notation" => "notation.md",
-        "Library" => [
-            "Public" => "lib/public.md",
-            "Internals" => "lib/internals.md",
-            "Differentiation" => "lib/differentiation.md",
-            "Testing" => "lib/testing.md",
+        "Features on Manifolds" => [
+            "Differentiation" => "features/differentiation.md",
+            "Distributions" => "features/distributions.md",
+            "Statistics" => "features/statistics.md",
+            "Utilities" => "features/utilities.md",
+            "Testing" => "features/testing.md",
```

Comment:
Maybe we could order these alphabetically? That is put "Testing" before "Utilities".

Reply:
Of course, I was just a little too fast.